### DISCUSSION TITLE
I have used a FluentIcon to display a column value as a tootip in a Datagrid

### DISCUSSION THREAD

**Discussion by jecojacob** [microsoft/fluentui-blazor#2753] [General]
I have used a FluentIcon to display a column value as a tooltip in a Datagrid under certain condition. Tooltip is shown only for the first row , and It shows values for all other columns and keeps flickering. Is it a known issue ? Any suggestions

**Comment by vnbaaij** [comment]
Besides thinking it is not a really good idea to do that, I think it doesn't work because you would need to have a @key on each tooltip when using any component in there. Blazor needs it to keep track of what to render where.
